Assets/HoloToolkit/Input/Prefabs/FocusedObjectKeywordManager.prefab

```diff
@@ -20,7 +20,6 @@ GameObject:
   m_Component:
```

Comment:
I agree.  But we should address that there are currently 4 meta files that keep getting regenerated because they were not properly added in the main branch:

two .fbm/templates.meta files for the cursor
Assets/HoloToolkit/Input/Plugins.meta
Assets/HoloToolkit/Scripts/Microphone.meta